autogen_device_driver.py

```diff
@@ -0,0 +1,40 @@
+import argparse
+import sys
+import logging
+
+from device_drivers import Generator as DeviceDriverGenerator, DriverConfig
```

Comment:
I've never seen an import statement like this. Just curious what exactly is happening here.

Reply:
So in `device_drivers/__init__.py` I have already imported these modules to make them available by just importing `device_drivers` and then I decided to rename `Generator` to `DeviceDriverGenerator` because within `device_drivers` it doesn't need the distinction but if we end up with multiple `Generator` modules, its clear in the code below that they are not the same module.

Reply:
Gotcha, that makes sense. I'm not used to having multiple imports on the same line, so that's what confused me. It's equivalent to 
```python
from device_drivers import Generator as DeviceDriverGenerator
from device_drivers import DriverConfig
```